sdk/core/ci.yml

```diff
@@ -1,11 +1,19 @@
 # DO NOT EDIT THIS FILE
 # This file is generated automatically and any changes will be lost.
 
+resources:
```

Comment:
Can this be added in the template or does it have to be added to each root level yml?

Reply:
Has to be at the root unfortunately.